src/man/sssd-krb5.5.xml

```diff
@@ -501,6 +501,32 @@
                     </listitem>
                 </varlistentry>
 
+                <varlistentry>
+                    <term>krb5_kdcinfo_lookahead (string)</term>
+                    <listitem>
+                        <para>
+                            When krb5_use_kdcinfo is set to true, you can limit amount
+                            of servers handed to sssd_krb5_locator_plugin. This might
+                            be helpfull when there are too many servers discovered
+                            using SRV record.
+                        </para>
+                        <para>
+                            The krb5_kdcinfo_lookahead option contains a two
+                            numbers seperated by colon. First number represents
+                            number of primary servers used and the second number
+                            specifies the number of backup servers.
+                        </para>
+                        <para>
+                            For example <emphasis>10:0</emphasis> means that up to
+                            10 primary servers will be handed to sssd_krb5_locator_plugin
+                            but no backup servers.
+                        </para>
+                        <para>
+                            Default: Use all servers
```

Comment:
I don't think we want to use all servers as a default. We want either 0 as the default to not cause a change in a released version or some very strict default like 3:1

Reply:
btw I would prefer some strict default larger than 0, but definitely not all